client/app/components/auth/signup.html

```diff
@@ -0,0 +1,14 @@
+<div class="authDiv">
+  <h1>Sign Up</h1>
+    Username:
+    <br>
+    <input type='text' ng-model="userCredentials.username">
+    <br>
+    Password:
+    <br>
+    <input type="text" ng-model="userCredentials.password">
+    <br>
+    <button ng-click="signUp(userCredentials)">SignUp</button>
```

Comment:
This `signUp` function doesn't take an argument. It uses `$scope.userCredentials` internally.